WordPress/Classes/ViewRelated/Reader/ReaderCardsStreamViewController.swift

```diff
@@ -144,6 +149,11 @@ class ReaderCardsStreamViewController: ReaderStreamViewController {
         controller.readerTopic = topic
         return controller
     }
+
+    /// Observe the managedObjectContext for changes (likes, saves) and reload the tableView
+    private func observeDisplayContext() {
+        NotificationCenter.default.addObserver(self, selector: #selector(reload(_:)), name: NSNotification.Name.NSManagedObjectContextDidSave, object: managedObjectContext())
```

Comment:
No need to change, but you could remove the `NSNotification.Name`

```suggestion
        NotificationCenter.default.addObserver(self, selector: #selector(reload(_:)), name: .NSManagedObjectContextDidSave, object: managedObjectContext())
```